src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const averageAgeMan = people
+    .filter((person) => century ? person.sex === 'm'
+    && Math.ceil(person.died / 100) === century : person.sex === 'm')
+    .map((getAge) => getAge.died - getAge.born);
+
+  const average = averageAgeMan
+    .reduce((sum, age) => sum + age / averageAgeMan.length, 0);
+
+  return +average.toFixed(2);
```

Comment:
average age calculation is duplicated, so please move it to the separate function and reuse it